Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/TransformTool.java

```diff
@@ -88,7 +88,7 @@ public TransformTool(TransformToolOptionsView transformToolOptionsView, final Co
 
 		cropAlgorithm = new JavaCropAlgorithm();
 
-		resetScaleAndTranslation();
+//		resetScaleAndTranslation();
```

Comment:
delete line if not needed
in: Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/TransformTool.java
`//		resetScaleAndTranslation();`